conjur/config.py

```diff
@@ -52,7 +54,6 @@ def __init__(self, config_file=DEFAULT_CONFIG_FILE, netrc_file=DEFAULT_NETRC_FIL
 
             setattr(self, attribute_name, config[config_field_name])
             self._config[attribute_name] = getattr(self, attribute_name)
-
```

Comment:
python is a dangerous language, incorrect tabbing can cause a nightmare bug.
I think you should always separate with new line after scope closing.
So don't delete this newline.